app/src/main/res/resource/values-de/strings.xml

```diff
@@ -154,30 +156,30 @@
     <string name="feedback_exceed_rate_limit">Ups! Es gab zu viele Besucher in dieser Stunde, bitte versuchen Sie in der nächsten Stunde erneut.</string>
 
     <string name="feedback_confirm_change_theme">Themaänderung bestätigen</string>
-    <string name="feedback_change_theme_warning">Jetztiges Ändern des Themas wird aktuelle downloads abbrechen.</string>
+    <string name="feedback_change_theme_warning">Änderung des Themas wird alle aktuelle Downloads abbrechen.</string>
     <string name="feedback_confirm_restart">Neustart bestätigen</string>
-    <string name="feedback_restart_warning">Jetztiges Neustarten der App wird alle Downloads abbrechen.</string>
+    <string name="feedback_restart_warning">Neustarten der App wird alle aktuellen Downloads abbrechen.</string>
 
     <string name="feedback_click_again_to_exit">Zum Verlassen nochmal drücken.</string>
 
     <string name="feedback_share_photo_title">Ein Foto von Unsplash.com</string>
-    <string name="feedback_share_photo_extra">Genommen von # am #</string>
+    <string name="feedback_share_photo_extra">Genommen von # am #\n</string>
```

Comment:
When the first # is the author and the second the date, this sounds strange. "Genommen von H.P.X am 17.05.2017"
Maybe use "Aufgenommen", which could be the correct translation as "the photo was taken by XY at Z".